### PR TITLE
[IMP] Improved Printouts Headers  - #259

### PR DESCRIPTION
**Pull request by YogeshMahera-SerpentCS**
[Business Requirement Report.pdf](https://github.com/OCA/business-requirement/files/1354996/Business.Requirement.Report.pdf)

### CODE REVIEW

**Comment by elicoidal** [comment]
#259

**Comment by elicoidal** [comment]
Almost there @YogeshMahera-SerpentCS :
* Source: should be 2 columns wide
* Can you reduce the space between Business Requirement Document and Name?

Please make sure to change the 4 reports

**Comment by YogeshMahera-SerpentCS** [comment]
@elicoidal  Improved it

**Comment by YogeshMahera-SerpentCS** [comment]
BR [Business Requirement Report.pdf](https://github.com/OCA/business-requirement/files/1355204/Business.Requirement.Report.pdf)
BRD [Business Requirement Deliverable Report.pdf](https://github.com/OCA/business-requirement/files/1355205/Business.Requirement.Deliverable.Report.pdf)
BRDR [Business Requirement Deliverable Resource Report.pdf](https://github.com/OCA/business-requirement/files/1355206/Business.Requirement.Deliverable.Resource.Report.pdf)

File: business_requirement/models/business.py [hunk 1]
```diff
@@ -290,6 +290,17 @@ def write(self, vals):
                     raise ValidationError(_(
                         'You can only move to the following stage: '
                         'draft/confirmed /cancel/drop.'))
+            if vals['state'] == 'stakeholder_approval':
```
Comment (elicoidal) [inline business_requirement/models/business.py:293]: Not sure why you have this modification here

**Review by elicoidal — APPROVED** [review]
Tested and LGTM

**Comment by elicoidal** [comment]
@sudhir-serpentcs can you review?

File: business_requirement/models/business.py (hunk 1 above)
Comment (YogeshMahera-SerpentCS) [reply business_requirement/models/business.py:293]: In #259 
>  Approval date should be when the SOA state is changed

That's why I have added this code here

**Comment by sudhir-serpentcs** [comment]
br_report

Small layout changes. Other than LGTM
@YogeshMahera-SerpentCS 
CC: @elicoidal

**Comment by YogeshMahera-SerpentCS** [comment]
@elicoidal @sudhir-serpentcs Improved changes

**Comment by YogeshMahera-SerpentCS** [comment]
Fixed issue of #254 

BRD [Business Requirement Deliverable Report.pdf](https://github.com/OCA/business-requirement/files/1361741/Business.Requirement.Deliverable.Report.pdf)
BRDR [Business Requirement Deliverable Resource Report.pdf](https://github.com/OCA/business-requirement/files/1361742/Business.Requirement.Deliverable.Resource.Report.pdf)

@elicoidal

**Comment by elicoidal** [comment]
@YogeshMahera-SerpentCS I would suggest 
- [x] to underline as well "Deliverables" titles

**Comment by elicoidal** [comment]
- [x] Report Other resources: not correct for the sequence  for RL and DL

Rest is good and tested

**Comment by YogeshMahera-SerpentCS** [comment]
@elicoidal Could you please review it

**Review by elicoidal — APPROVED** [review]
Functional tests OK

**Comment by elicoidal** [comment]
@victormartinelicocorp can you have a look?

**Review by martintorresvm — APPROVED** [review]
LGTM

**Pull request merged**